src/tools/pub/cmd/pub-add.md

```diff
@@ -38,6 +38,10 @@ $ dart pub add --dev test
 For options that apply to all pub commands, see
 [Global options](/tools/pub/cmd#global-options).
 
+The options given will apply to all the packages added in an invocation. For
+example `dart pub add test http --dev` will add both package:test and
+package:http as dev-dependencies.
```

Comment:
Some minor adjustments to be consistent with the rest of the page and make it a note to better fit with the flow of the page:

```suggestion
{{site.alert.note}}
  Any specified options will apply to all the packages
  included in an invocation of the command.
  For example, `dart pub add test http --dev` 
  will add both the `test` and `http` packages 
  as dev dependencies.
{{site.alert.end}}
```